bitstore/controllers.py

```diff
@@ -51,6 +51,8 @@ def authorize(auth_token, req_payload):
             })
             try:
                 s3path = config['STORAGE_PATH_PATTERN'].format(**format_params)
+                # Avoid double slash if "dataset" is not in req_payload (as it is optional)
```

Comment:
@rufuspollock if it is optional, it may not be there. so if 
`STORAGE_PATH_PATTERN='path/{owner}/{dataset}/datapackage.json'`
`s3path` will end up to be `path/owner//datapackage.json` no